packages/@pollyjs/ember/index.js

```diff
@@ -75,7 +77,7 @@ module.exports = {
       server: {}
     };
 
-    let configPath = path.join(this.root, 'config', 'polly.js');
+    let configPath = path.join(root, 'config', 'polly.js');
```

Comment:
What are your thoughts on doing something like

`const projectRoot = this.project && this.project.root ? this.project.root : root`?

Reply:
I like it, will update

Reply:
Updated